src/w3c/templates/headers.js

```diff
@@ -0,0 +1,159 @@
+import "deps/hyperhtml";
+import showLogo from "./show-logo";
+import showPeople from "./show-people";
+
+export default conf => {
+  const html = hyperHTML;
+  return html`<div class='head'>
+  ${conf.logos.map(showLogo)}
+  <h1 class='title p-name' id='title' property='${conf.doRDFa ? "dcterms:title" : null}'>${conf.title}</h1>
+  ${conf.subtitle ? html`
+    <h2 property='${conf.doRDFa ? "bibo:subtitle" : null}' id='subtitle'>${conf.subtitle}</h2>
+  ` : ""}
+  <h2>${conf.prependW3C ? `W3C ` : ""}${conf.textStatus} <time property='${conf.doRDFa ? "dcterms:issued" : null}' class='dt-published' datetime='${conf.dashDate}'>${conf.publishHumanDate}</time></h2>
+  <dl>
+    ${!conf.isNoTrack ? html`
+      <dt>${conf.l10n.this_version}</dt>
+      <dd><a class='u-url' href='${conf.thisVersion}'>${conf.thisVersion}</a></dd>
+      <dt>${conf.l10n.latest_published_version}</dt>
+      <dd>${conf.latestVersion ? html`<a href='${conf.latestVersion}'>${conf.latestVersion}</a>` : "none"}</dd>
+    ` : ""}
+    ${conf.edDraftURI ? html`
+      <dt>${conf.l10n.latest_editors_draft}</dt>
+      <dd><a href='${conf.edDraftURI}'>${conf.edDraftURI}</a></dd>
+    ` : ""}
+    ${conf.testSuiteURI ? html`
+      <dt>Test suite:</dt>
+      <dd><a href='${conf.testSuiteURI}'>${conf.testSuiteURI}</a></dd>
+    ` : ""}
+    ${conf.implementationReportURI ? html`
+      <dt>Implementation report:</dt>
+      <dd><a href='${conf.implementationReportURI}'>${conf.implementationReportURI}</a></dd>
+    ` : ""}
+    ${conf.bugTrackerHTML ? html`
+      <dt>${conf.l10n.bug_tracker}</dt>
+      <dd>${[conf.bugTrackerHTML]}</dd>
+    ` : ""}
+    ${conf.isED ? html`
+      ${conf.prevED ? html`
+        <dt>Previous editor's draft:</dt>
+        <dd><a href='${conf.prevED}'>${conf.prevED}</a></dd>
+      ` : ""}
+    ` : ""}
+    ${conf.showPreviousVersion ? html`
+      <dt>Previous version:</dt>
+      <dd><a rel='${conf.doRDFa ? "dcterms:replaces" : null}' href='${conf.prevVersion}'>${conf.prevVersion}</a></dd>
+    ` : ""}
+    ${conf.prevRecURI ? html`
+      ${conf.isRec ? html`
+          <dt>Previous Recommendation:</dt>
+          <dd><a rel='${conf.doRDFa ? "dcterms:replaces" : null}' href='${conf.prevRecURI}'>${conf.prevRecURI}</a></dd>
+      ` : html`
+          <dt>Latest Recommendation:</dt>
+          <dd><a href='${conf.prevRecURI}'>${conf.prevRecURI}</a></dd>
+      `}
+    ` : ""}
+    <dt>${conf.multipleEditors ? html`${conf.l10n.editors}` : html`${conf.l10n.editor}`}</dt>
+    ${showPeople(conf, "Editor", conf.editors)}
+    ${conf.authors ? html`
+      <dt>${conf.multipleAuthors ? [conf.l10n.authors] : [conf.l10n.author]}</dt>
+      ${showPeople(conf, "Author", conf.authors)}
+    ` : ""}
+    ${conf.otherLinks ? html`
+      ${conf.otherLinks.map(link => html`
```

Comment:
This one seems a little hard to work with. Might be worth slitting into its own function?